### PR TITLE
MVC Component Header 

### LINKED ISSUE #20: MVC component: Header
image

### PR DESCRIPTION
**Pull request by Anastasiia0999**
image

- Click on LOGO - cars` gallery render( check publisher event)

- Click on Partners Link - partners render( check publisher event)

- Click on Contacts Link - contacts render( check publisher event)

- Input for search of a car`s model

### CODE REVIEW

File: app/components/header/header-view.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
+import './header-styles.scss';
+
+export class HeaderView {
+    headerElem = document.querySelector('.header');
+
+    constructor({ contacts, partners, home, search }) {
+      this.contacts = contacts;
+      this.partners = partners;
+      this.home = home;
```
Comment (PavelKundenko) [inline app/components/header/header-view.js]: I think that it's better to name these handlers in a more explicit way, for example `this.handleContactsClick`

**Review by PavelKundenko — APPROVED** [review]

**Review by Winchestery — APPROVED** [review]

**Review by MarinaPasternak — APPROVED** [review]

File: app/components/header/header-view.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+                   <form class="form-inline">
+                       <input 
+                           type="text" 
+                           class="form-control header__input " 
+                           placeholder="Search" 
+                           aria-label="Username" 
+                           aria-describedby="basic-add">
+                      <div class="input-group-prepend search-btn">
+                        <span class="input-group-text" id="basic-add">
+                            <svg width="1em" height="1em" viewBox="0 0 16 16" class="bi bi-search" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
```
Comment (so2niko) [inline app/components/header/header-view.js]: Save svg in file and connect here in img or something else

**Review by so2niko — CHANGES_REQUESTED** [review]
Check, please,  situation with svg file

**Review by MozdolevskyiMaksym — APPROVED** [review]

File: app/components/header/header-view.js (hunk 2 above)
Comment (Anastasiia0999) [reply app/components/header/header-view.js]: done

**Review by so2niko — APPROVED** [review]

**Pull request merged**